### DISCUSSION TITLE
move 或者copy 文件无效？

### DISCUSSION THREAD

**Discussion by mumu157** [houtianze/bypy#647] [Q&A]
想通过mv指令移动文件，但是没有效果，是有bug还是有啥步骤问题？

如下所示，无论是mv还是copy，最终test文件下都没有test.log

```
mumu@Ubuntu2004:~/codes/MacroPlacement$ bypy mv test.log test -v
Loading Hash Cache File '/home/mumu/.bypy/bypy.hashcache.json'...
Hash Cache File loaded.
Skip saving Hash Cache since it has not been updated.

mumu@Ubuntu2004:~/codes/MacroPlacement$ bypy cp test.log test -v
Loading Hash Cache File '/home/mumu/.bypy/bypy.hashcache.json'...
Hash Cache File loaded.
Skip saving Hash Cache since it has not been updated.

mumu@Ubuntu2004:~/codes/MacroPlacement$ bypy list
/apps/bypy ($t $f $s $m $d):
D benchmark_DREMPLace 0 2023-06-27, 12:09:54
D dac2012 0 2023-06-27, 11:36:06
D iccad2015.ot 0 2023-06-27, 10:43:30
D ispd2005 0 2023-06-27, 10:50:16
D ispd2015 0 2023-06-27, 09:38:49
D test 0 2023-06-27, 12:17:34
F .gitignore 18 2023-06-27, 09:35:43 9c3182328hd014d62b5df367eab43070
F CMakeLists.txt 188 2023-06-27, 09:35:45 b42d35407q9802a0fdc589ddbba8d975
F download_dac2012.sh 266 2023-06-27, 09:35:46 beb813a72l5c37dda8624442b6428f17
F iccad2015.ot.md 179 2023-06-27, 09:35:47 427c9c1a5s38d4e89c9645f5408214bd
F ispd2005_2015.py 989 2023-06-27, 12:14:52 d75448a92n165372eef7ea3086ddb10a
F test.log 130 2023-06-27, 12:31:20 b502f7370kca31852104334115058dc2

mumu@Ubuntu2004:~/codes/MacroPlacement$ bypy list test
/apps/bypy/test ($t $f $s $m $d):
F README.md 43348 2023-06-27, 12:17:34 52da151afgbb4f0ea3ea25ee37ae1bec
mumu@Ubuntu2004:~/codes/MacroPlacement$ 
```

**Comment by houtianze** [comment]
没看代码，你可以试试这个命令：`my -v test.log test/`是否可以？On 27 Jun 2023, at 12:36 PM, Lee Zhang ***@***.***> wrote:﻿
想通过mv指令移动文件，但是没有效果，是有bug还是有啥步骤问题？
如下所示，无论是mv还是copy，最终test文件下都没有test.log
***@***.***:~/codes/MacroPlacement$ bypy mv test.log test -v
Loading Hash Cache File '/home/mumu/.bypy/bypy.hashcache.json'...
Hash Cache File loaded.
Skip saving Hash Cache since it has not been updated.

***@***.***:~/codes/MacroPlacement$ bypy cp test.log test -v
Loading Hash Cache File '/home/mumu/.bypy/bypy.hashcache.json'...
Hash Cache File loaded.
Skip saving Hash Cache since it has not been updated.

***@***.***:~/codes/MacroPlacement$ bypy list
/apps/bypy ($t $f $s $m $d):
D benchmark_DREMPLace 0 2023-06-27, 12:09:54
D dac2012 0 2023-06-27, 11:36:06
D iccad2015.ot 0 2023-06-27, 10:43:30
D ispd2005 0 2023-06-27, 10:50:16
D ispd2015 0 2023-06-27, 09:38:49
D test 0 2023-06-27, 12:17:34
F .gitignore 18 2023-06-27, 09:35:43 9c3182328hd014d62b5df367eab43070
F CMakeLists.txt 188 2023-06-27, 09:35:45 b42d35407q9802a0fdc589ddbba8d975
F download_dac2012.sh 266 2023-06-27, 09:35:46 beb813a72l5c37dda8624442b6428f17
F iccad2015.ot.md 179 2023-06-27, 09:35:47 427c9c1a5s38d4e89c9645f5408214bd
F ispd2005_2015.py 989 2023-06-27, 12:14:52 d75448a92n165372eef7ea3086ddb10a
F test.log 130 2023-06-27, 12:31:20 b502f7370kca31852104334115058dc2

***@***.***:~/codes/MacroPlacement$ bypy list test
/apps/bypy/test ($t $f $s $m $d):
F README.md 43348 2023-06-27, 12:17:34 52da151afgbb4f0ea3ea25ee37ae1bec
***@***.***:~/codes/MacroPlacement$ bypy cp test.log test/ -v
Loading Hash Cache File '/home/mumu/.bypy/bypy.hashcache.json'...
Hash Cache File loaded.
Skip saving Hash Cache since it has not been updated.

**Reply by mumu157** [reply]
试过了也不行~

**Comment by houtianze** [comment]
贴下输出？On 27 Jun 2023, at 2:37 PM, Lee Zhang ***@***.***> wrote:﻿
试过了也不行~
